components/automate-cli/pkg/verifyserver/models/batchcheck.go

```diff
@@ -152,7 +152,15 @@ func (c *Config) PopulateWith(haConfig *config.HaDeployConfig) error {
 		c.populateConfigInitials(haConfig)
 	}
 
-	if haConfig.GetObjectStorageConfig() != nil {
+	if haConfig.GetConfigInitials().BackupConfig == "file_system" {
+		c.Backup = &Backup{
+			FileSystem: &FileSystem{
+				MountLocation: haConfig.GetConfigInitials().BackupMount,
+			},
+		}
+	}
+
+	if haConfig.GetObjectStorageConfig() != nil && haConfig.GetConfigInitials().BackupConfig == "object_storage" {
```

Comment:
I think you also also check if `c.Backup = nil`

Reply:
Done